ios/FluentUI/Other Cells/ActionsCell.swift

```diff
@@ -202,13 +244,13 @@ open class ActionsCell: UITableViewCell {
     open override func setSelected(_ selected: Bool, animated: Bool) { }
 
     private func setupAction(_ button: UIButton) {
-        button.titleLabel?.font = TableViewCell.TextStyles.title.font
+        button.titleLabel?.font = UIFont.fluent(tokens.titleFont)
```

Comment:
any similar issues like you did in tableviewcell update which the font size and weight  is different from fluent1 ?

Reply:
None, but also those only affected fonts that were not the title font (subtitle, footer, etc)